### PR TITLE
Spec for supporting Android App Bundles in App Center.

### PR DESCRIPTION
**Pull request by dimazaid**

### CODE REVIEW

File: specs/aab-store and build/aab.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# App Center Stores and Build support for Android App Bundles
+	
+This proposal will cover a new feature to to add support for Android App Bundles in App Center.
+This feature will only cover support for .aab packages in the Stores service, and Build.
+	
+## Problem Statement
+	
+Google has introduced AAB and it's been the recommended method to upload packages to the Play Store.
+
+Google has also announced that By August 1, 2019, all apps that use native code must provide a 64-bit version in addition to the 32-bit version in order to publish an update.
```
Comment (Oddj0b) [inline specs/aab-store and build/aab.md]: Maybe make it clear that with `.aab` developers get the required 64-bit version.

File: specs/aab-store and build/aab.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	
+## Solution
+	
+### Build
+* Given an Android project where the configuration contains a bundle, then users will see a new setting under build configuration for building an Android Bundle package.
+
+* App Center will offer a new selector in the Android build configuration: "Also Build an Android Bundle".
+This will be pre-selected if the module selected in the configuration screen contains a "bundle {...}" block.
+
+* If a bundle will be built, the build will have the "android-app-bundle" tag after it is finished.
```
Comment (DennisPan) [inline specs/aab-store and build/aab.md]: This is implementation detail. Let's remove it

File: specs/aab-store and build/aab.md (hunk 1 above)
Comment (dimazaid) [reply specs/aab-store and build/aab.md]: Added.

File: specs/aab-store and build/aab.md (hunk 2 above)
Comment (dimazaid) [reply specs/aab-store and build/aab.md]: Removed.

**Comment by sseyod** [comment]
Will this work with AppCenter NDK-based crash reporting for Android?

i.e. https://docs.microsoft.com/en-us/appcenter/diagnostics/android-ndk

Thanks, Pete

**Comment by dimazaid** [comment]
> Will this work with AppCenter NDK-based crash reporting for Android?
> 
> i.e. https://docs.microsoft.com/en-us/appcenter/diagnostics/android-ndk
> 
> Thanks, Pete

Hi Pete,
Sorry I missed your message last week. Is your question whether app bundles will be supported in crashes?

**Comment by sseyod** [comment]
Hi @dimazaid, yes, but more specifically, that the NDK-side of crash reporting at the C++ level will be supported. Thanks, Pete

**Comment by dimazaid** [comment]
Thanks, Pete. For now, this will only support distribution to store and build. We'll look into supporting crashes at a later time.

**Pull request merged**